surface/secretsmanager/tests/test_commands/test_import_secrets.py

```diff
@@ -0,0 +1,88 @@
+import json
+import tempfile
+from io import StringIO
+from django.test import TestCase
+from django.core.management import call_command
+from secretsmanager.models import Secret, SecretLocation
+from inventory.models import GitSource
+
+class ImportSecretsCommandTest(TestCase):
```

Comment:
The assertions using TestCase are legacy and we'd rather use Pytest's assertion form now. Can you please fix it here?